.github/workflows/additional-ci-image-checks.yml

```diff
@@ -84,6 +84,10 @@ on:  # yamllint disable-line rule:truthy
         description: "Whether to debug resources (true/false)"
         required: true
         type: string
+      use-uv:
+        description: "Whether to use uv to build the image (true/false)"
+        required: true
```

Comment:
Does it need to be mandatory? We can use it by default and then fall back if not set right?

Reply:
All other inputs are mandatory. The intent seems to be clarity by being explicit.

Reply:
Yes @rawwar . And also validation. When we have mandatory input and we forget to add it, we got "workflow is broken" error, and this forces us to think about those and make sure that we do not have "accidental" values passed to compoiste workflows.